### PR TITLE
docs: Updated doc to include OVMS info

### PR DESCRIPTION
**Pull request by vli11**
Closes: #65

## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [x] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?
Updated doc to include gRPC (OVMS) info

## Issue this PR will close

close: #65 

## Test Instructions if applicable
you should see OVMS folder under docs_src directory, this OVMS folder includes OVMS pipeline setup and pipeline run documentation.

### CODE REVIEW

File: docs_src/OVMS/pipelinesetup.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Set up Pipeline
+
+### Step 1: Clone the repository
+
+```
+git clone  https://github.com/intel-retail/automated-self-checkout.git && cd ./automated-self-checkout
+```
+
+### Step 2: Build the benchmark containers (coming up)
```
Comment (jim-wang-intel) [inline docs_src/OVMS/pipelinesetup.md:9]:
```suggestion
### Step 2: Build the benchmark images (coming up)
```

File: docs_src/OVMS/pipelinesetup.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+For use cases that support benchmarking, build the benchmark containers:
```
Comment (jim-wang-intel) [inline docs_src/OVMS/pipelinesetup.md:11]:
```suggestion
For use cases that support benchmarking, build the benchmark Docker images:
```

File: docs_src/OVMS/pipelinesetup.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+|                                   |                                    | docker images command to show <b>openvino/model_server:latest-gpu</b>  |
+|                                   |                                    | docker images command to show <b>openvino/model_server:latest</b>      |
+|                                   |                                    | docker images command to show <b>openvino/model_server-pkg:latest</b>  |
+|                                   |                                    | docker images command to show <b>openvino/model_server-build:latest</b>|
+| OVMS Client                       | <pre>make build-ovms-client</pre>  | docker images command to show <b>ovms-client:latest</b>                |
+
+**_Note:_** Build command may take a while, depending on your internet connection and machine specifications.
+
+**_Note:_** If the build command succeeds, you will see all the built Docker images files as indicated in the **Check Success** column. If the build fails, check the console output for errors. The dependencies might have been unable to resolve. Address the issue and repeat [from step 3](/pipelinesetup.md#step-3).
```
Comment (jim-wang-intel) [inline docs_src/OVMS/pipelinesetup.md:69]: i don't understand this note... why docker image build failure has anything to do with model downloading ?

File: docs_src/OVMS/pipelinerun.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+## Prerequisites 
+Before running, [set up the pipeline](./pipelinesetup.md).
+
+---
+## Overview 
+When the pipeline is run, the `docker-run.sh` script starts the service and performs inferencing on the selected input media. The output of running the pipeline provides the inference results for each frame based on the media source such as text, barcode, and so on, as well as the frames per second (FPS). Pipeline run provides many options in media type, system process platform type, and additional optional parameters. These options give you the opportunity to compare what system process platform is better for your need.
+
+## Start Pipeline
+
+You can run the pipeline script, `docker-run.sh`, with the following input parameters:
```
Comment (jim-wang-intel) [inline docs_src/OVMS/pipelinerun.md:12]: i think it is important to mention when ovms docker run to use --workload parameter explicitly for ovms as the default for that is dlstreamer.

File: docs_src/OVMS/pipelinerun.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+| ovms-client:latest                                 | Exited (0) 29 seconds ago    | ovms-client0 |
+| openvino/model_server-gpu:latest                   | Up 59 seconds                | model-server |
+
+
+Check inference results and use case performance
+```bash
+ls -l results
+```
+
+The **results** directory contains maskrcnn.log
```
Comment (jim-wang-intel) [inline docs_src/OVMS/pipelinerun.md:100]:
```suggestion
The **results** directory should contain maskrcnn.log
```

**Review by jim-wang-intel — CHANGES_REQUESTED** [review]

File: docs_src/OVMS/pipelinesetup.md (hunk 3 above)
Comment (vli11) [reply docs_src/OVMS/pipelinesetup.md:69]: build can fail if the model downloading failed, so this is asking user to check from step 3 (download models)

Comment (jim-wang-intel) [reply docs_src/OVMS/pipelinesetup.md:69]: could you show me?

File: docs_src/OVMS/pipelinesetup.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
+# Set up Pipeline
+
+### Step 1: Clone the repository
+
+```
+git clone  https://github.com/intel-retail/automated-self-checkout.git && cd ./automated-self-checkout
+```
+
+### Step 2: Build the benchmark images (coming up)
```
Comment (jim-wang-intel) [inline docs_src/OVMS/pipelinesetup.md:9]: I prefer to call out the Docker images as there is also picture `image` we are downloading as part of steps.

```suggestion
### Step 2: Build the benchmark Docker images (coming up)
```

File: docs_src/OVMS/pipelinesetup.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+For use cases that support benchmarking, build the benchmark images:
```
Comment (jim-wang-intel) [inline docs_src/OVMS/pipelinesetup.md:11]:
```suggestion
For use cases that support benchmarking, build the benchmark Docker images:
```

**Review by jim-wang-intel — APPROVED** [review]
LGTM

File: docs_src/OVMS/pipelinesetup.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+
+### Step 4: Download image file Manually (Optional)
+
+The sample image downloader script is automatically called as part of docker-run-opencv-ovms.sh. You can also download the sample image manually using script below:
+
+```bash
+sh ./configs/opencv-ovms/scripts/image_download.sh 
+```
+
+**_Note:_** Depending on your internet connection, this might take less than a minute.
```
Comment (marcpfuller) [inline docs_src/OVMS/pipelinesetup.md:37]:
```suggestion
**_Note:_** Depending on your internet connection, this might take longer than a minute.
```

**Pull request merged**